src/main/scala/beam/agentsim/agents/modalBehaviors/ChoosesMode.scala

```diff
@@ -72,7 +72,7 @@ trait ChoosesMode extends BeamAgent[PersonData] with TriggerShortcuts with HasSe
     case Event(TriggerWithId(FinalizeModeChoiceTrigger(tick), theTriggerId), info: BeamAgentInfo[PersonData]) =>
       hasReceivedCompleteChoiceTrigger = true
       val result : State = completeChoiceIfReady(Some(theTriggerId))
-      if(result.stateName == ChoosesMode){
+      if(result.stateName == ChoosingMode){
```

Comment:
@colinsheppard There is an attempt to compare unrelated types. Probably, misprint